pkg/descriptions.json

```diff
@@ -2266,5 +2266,10 @@
     "Name": "Microsoft.IO.Redist",
     "Description": "Downlevel support package for System.IO classes.",
     "CommonTypes": []
+  },
+  {
+    "Name": "Microsoft.Reflection.MetataLoadContext",
+    "Description": "Provides functionality to reflect on assemblies with different architectures and without interferring with AppDomain contexts.",
```

Comment:
typo "interfering"

AppDomain contexts are mentioned because the package supports desktop as well?

Reply:
How about this:
```Provides read-only reflection on assemblies in an isolated context with support for assemblies that have different architectures and runtimes```